### PR TITLE
Add Config Files for OCP 4.7 over BareMetal

### PR DESCRIPTION
**Pull request by prsurve**
Signed-off-by: Pratik Surve <prsurve@redhat.com>

### CODE REVIEW

**Review by ebenahar — DISMISSED** [review]

**Comment by ebenahar** [comment]
@prsurve do we have an issue that this PR fixes?

File: conf/ocsci/ocp_47_bm_workaround.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DEPLOYMENT:
```
Comment (ebenahar) [inline conf/ocsci/ocp_47_bm_workaround.yaml:1]: is this is a file needed for a workaround? if so, please open a w/a issue

Comment (prsurve) [reply conf/ocsci/ocp_47_bm_workaround.yaml:1]: yes for now it's a WA

**Comment by prsurve** [comment]
> @prsurve do we have an issue that this PR fixes?

Yes OCP installation is failing I'm trying to check why it failed

**Comment by prsurve** [comment]
Closing this pr will create a new one

**Pull request closed without merging**